diff_drive_controller/CMakeLists.txt

```diff
@@ -3,13 +3,14 @@ project(diff_drive_controller)
 
 find_package(catkin REQUIRED COMPONENTS
     controller_interface
-    urdf
+    nav_msgs
     realtime_tools
     tf
-    nav_msgs)
+    urdf
+)
 
 catkin_package(
-  INCLUDE_DIRS include
```

Comment:
In our fork (https://github.com/clearpathrobotics/ros_controllers) we're actually using it as a library to provide more complex and lower level velocity limits, like the ones enforce by a safety LIDAR.

So I wouldn't remove this line.